policies/release-requirements.md

```diff
@@ -0,0 +1,117 @@
+Release Requirements Policy
+===========================
+
+The OpenSSL project team creates the following three types of OpenSSL software
+releases:
+
+- [stable] releases
+- [alpha] (pre-)releases
+- [beta] (pre-)releases
+
+This policy defines the requirements on the state of a branch in the source
+tree that must be met before a release from that branch can be done.
+
+Alpha Pre-releases
+------------------
+
+As this is just a preview release for testing things that have been worked
+on in the development branch, the requirements are minimal.
+
+- The CI must pass on the tip of the development branch before the release
+  commits were added to the tree.
+
+Beta Pre-releases
+-----------------
+
+The API and ABI should be stable and the source code should be feature complete
+by the first beta pre-release. The following release requirements apply to beta
+pre-releases.
+
+- The code is functionaly complete in regards to the particular release
```

Comment:
There is a typo: s/functionaly/functionally

Reply:
fixed